packages/subgraph/subgraph.yaml

```diff
@@ -131,3 +133,29 @@ templates:
       eventHandlers:
         - event: CollBalanceUpdated(indexed address,uint256)
           handler: handleCollSurplusBalanceUpdated
+  - name: LqtyStake
```

Comment:
I would call this LQTYStaking to be consistent with existing data sources, and to distinguish it from the entity that it creates. (I've been naming mappings modules according to the backend contract whose events they map).